.nvmrc

```diff
@@ -1 +1 @@
-8.9.0
+10
```

Comment:
I think we should move it to 14 or 16, as per https://nodejs.org/en/about/releases/

Reply:
Yes, I went very conservatively about this. Probably too much, bc 10 has passed away, and 12 is going to soon.

Reply:
Just updated to 16, and axios to 0.24.0, seems to run fine.